app/i18n/it/conf.php

```diff
@@ -149,6 +149,10 @@
 		'image_url' => 'Image URL',	// TODO
 		'name' => 'Nome',
 		'no_filter' => 'Nessun filtro',
+		'no_queries' => array(
+			'_' => 'No user queries are safed yet.',	// TODO
```

Comment:
```suggestion
			'_' => 'Nessuna ricerca personale è stata ancora inserita',
```
I had to edit the sentence a bit because different words were used in the Italian translation of the page.

I'll write you the translation from Italian to English below so you can understand why I had to change the sentence
` Personal research `

I write the translation of the modified sentence below
`No personal research has been entered yet`